receiver/googlecloudspannerreceiver/testdata/config.yaml

```diff
@@ -0,0 +1,44 @@
+receivers:
+  googlecloudspanner:
+    collection_interval: 10s
+    top_metrics_query_max_rows: 100
+    projects:
+      - project_id: "spanner project 1"
+        service_account_key: "path to spanner project 1 service account json key"
+        instances:
+          - id: "id1"
```

Comment:
For projects, you have project id while for instances you have id. Please make it symmetrical.

Reply:
Ok, will do.